scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -247,6 +247,9 @@ <h4 class="">
         {% if case.vcf_files.vcf_sv %}
           <a class="btn btn-dark btn-sm text-white" href="{{ url_for('variants.sv_variants', institute_id=institute._id, case_name=case.display_name, variant_type='clinical') }}">Clinical {% if case.track=="cancer" %}germline {% endif %}structural variants</a>
         {% endif %}
+        {% if case.vcf_files.vcf_mei %}
+          <a class="btn btn-dark btn-sm text-white" href="{{ url_for('variants.mei_variants', institute_id=institute._id, case_name=case.display_name, variant_type='clinical') }}">Clinical {% if case.track=="cancer" %}germline {% endif %}Mobile Element Insertion variants</a>
```

Comment:
why if case.track=="cancer" ??

Reply:
Just left in as it is now. At some point in the future, we will surely have this for the somatic side as well but let's remove it for now and add it back if needed.